dali/test/python/test_operator_jpeg_compression_distortion.py

```diff
@@ -24,47 +24,138 @@
 test_data_root = os.environ['DALI_EXTRA_PATH']
 images_dir = os.path.join(test_data_root, 'db', 'single', 'tiff')
 dump_images = False
+sequence_length = 10
+
+
+class InputImagesIter(object):
+    def __init__(self, batch_size, sequence_length):
+        self.batch_size = batch_size
+        self.sequence_length = sequence_length
+        with open(os.path.join(images_dir, 'image_list.txt')) as file:
+            self.files = [line.rstrip() for line in file if line != '']
+
+    def __iter__(self):
+        self.i = 0
+        return self
+
+    def __next__(self):
+        batch = []
+        for _ in range(self.batch_size):
+            in_img = None
+            # Skip input image if format isn't supported by OpenCV
+            while in_img is None:
+                filename, label = self.files[self.i].split(' ')
+                in_img = cv2.imread(os.path.join(images_dir, filename))
+                self.i = (self.i + 1) % len(self.files)
+            # Convert to rgb match dali channel order
+            rgb = cv2.cvtColor(in_img, cv2.COLOR_BGR2RGB)
+            batch.append(np.stack([rgb] * self.sequence_length))
+        return batch
+
+
+def _testimpl_jpeg_compression_distortion(batch_size, device, quality, format):
+    # TODO: JpegCompressionDistortion for 'FHWC' GPU format is not yet implemented
+    if device == 'cpu' and format == 'FHWC':  # TODO: remove device limitation
+        _testimpl_jpeg_compression_distortion_vid(batch_size, device, quality)
+    elif format != 'FHWC':
+        _testimpl_jpeg_compression_distortion_img(batch_size, device, quality)
+
+
+def _testimpl_jpeg_compression_distortion_vid(batch_size, device, quality):
+
+    @pipeline_def(batch_size=batch_size, num_threads=3, device_id=0)
+    def jpeg_distortion_pipe_vid(device='cpu', quality=None, ):
+        iii = InputImagesIter(batch_size, sequence_length)
+        in_seq = fn.external_source(source=iii, layout='FHWC')
```

Comment:
How about using our stack operator?
```suggestion
        encoded, _ = fn.readers.file(file_root=images_dir)
        in_images = fn.decoders.image(encoded, device='cpu')
        in_seq = fn.stack(*[images]*seq_len)
```

Reply:
Will try, sorry missed that comment